frontend/admin/.babelrc

```diff
@@ -2,6 +2,13 @@
   "plugins": [
     "@babel/plugin-proposal-class-properties",
     "@babel/plugin-proposal-private-methods"
+  ],
+  "presets": [
+    ["@babel/preset-env", {
+      "useBuiltIns":"usage",
```

Comment:
[Look's like](https://babeljs.io/docs/en/babel-preset-env#usebuiltins) for this to work, we also need to have core-js as a dependency in our package.json.

Reply:
I was actually following that Medium post you linked to earlier:
![image](https://user-images.githubusercontent.com/8978655/160682916-3e732350-ddcf-4b93-a12e-4beee50e85d4.png)
Are you sure about this?  I thought babel injected just the needed polyfills when compiling?

Reply:
Hmm, I read some more and saw stuff indicating it did need to be there?  Now I'm not sure....   I tried adding it and npm didn't actually add any packages so I think it's in our bundle already as a downstream dependency.  Do I left it in so at least it's explicit.

Reply:
Huh, you're right, the Medium post clearly says dev dependencies. But it also sounds like webpack injects "imports" to the files, not the full code, which suggests core-js would still need to be a production dependency... Yeah, I think leaving it in dependencies is a safe move.